protobuf/model_config.proto

```diff
@@ -1377,6 +1408,16 @@ message ModelSequenceBatching
   //@@     model.
   //@@
   repeated ControlInput control_input = 2;
+
+  //@@  .. cpp:var:: State state (repeated)
+  //@@
+  //@@     The implicit state that should be stored in Triton for performing
```

Comment:
From the language it appears we require the users to use implicit state management for performing inference requests in a sequence. I don't think that is right.. We must explicitly call the feature is optional.

Reply:
Will modify this to indicate that this is optional.